examples/gno.land/p/gov/dao/members.gno

```diff
@@ -0,0 +1,34 @@
+package dao
+
+import (
+	"std"
+)
+
+// MemberStore defines the member storage abstraction
+type MemberStore interface {
+	// GetMembers returns all members in the store
+	GetMembers() []Member
```

Comment:
Size()

Reply:
Added:


84aeb30

Reply:
Also added for the `PropStore`:

[43e5c66](https://github.com/gnolang/gno/pull/2581/commits/43e5c66e2ec8cdd9a977d995d395e2742413e139)